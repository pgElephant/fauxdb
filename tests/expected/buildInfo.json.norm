{
  "gitVersion": "fauxdb-1.0.0",
  "ok": 1,
  "version": "1.0.0"
}
