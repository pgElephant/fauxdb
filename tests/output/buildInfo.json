{
  ok: 1,
  version: '1.0.0',
  gitVersion: 'fauxdb-1.0.0',
  modules: 'none',
  allocator: 'system',
  javascriptEngine: 'none',
  sysInfo: 'FauxDB Server'
}
