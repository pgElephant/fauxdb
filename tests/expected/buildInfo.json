{
  "ok": 1,
  "version": "1.0.0",
  "gitVersion": "fauxdb-1.0.0"
}
